output.txt

```diff
@@ -0,0 +1,112 @@
+Running Release|Win32 interpreter...
```

Comment:
nit: Do we need this file? They typically go quickly out of sync with the code.

Reply:
Sure, I'll remove it and the test script as soon as we have unit tests. Which should be.. soon